src/BooleanFlag.php

```diff
@@ -1,27 +1,35 @@
 <?php
 
-namespace Sidekicker\Feature;
+namespace Sidekicker\FlagrFeature;
 
 use Flagr\Client\Api\FlagApi;
 use Flagr\Client\Api\TagApi;
 use Flagr\Client\Model\CreateFlagRequest;
 use Flagr\Client\Model\CreateTagRequest;
 use Flagr\Client\Model\Flag;
 
-class CreateFlag
+class BooleanFlag
```

Comment:
Ideally, we probably should have a FlagInterface? and have different types of implementations. But good enough for now.